### PR TITLE
Add an organization parameter to all billing methods

### PR DESCRIPTION
**Pull request by thgreasi**
Change-type: minor
Signed-off-by: Thodoris Greasidis <thodoris@balena.io>

---
##### Contributor checklist

- [ ] Includes tests
- [ ] Includes typings
- [ ] Includes updated documentation
- [ ] Includes updated build output

### CODE REVIEW

**Comment by thgreasi** [comment]
Since w/o the `if`s the UI would break when upgrading the SDK, I would like to stick with semver semantics.
I will drop all the `if`s right after and push a commit in the `v15` which sounds like it's going to happen quite soon.

File: DOCUMENTATION.md [hunk 1]
```diff
@@ -6045,11 +6050,16 @@ balena.models.billing.getAccount(function(error, billingAccount) {
 ```
 <a name="balena.models.billing.getPlan"></a>
 
-##### billing.getPlan() ⇒ <code>Bluebird</code>
+##### billing.getPlan([organization]) ⇒ <code>Bluebird</code>
```
Comment (sradevski) [inline DOCUMENTATION.md:6053]: Why is it in `[]`?

Comment (thgreasi) [reply DOCUMENTATION.md:6053]: Indeed that's how jsdoc marks optional arguments.
This is also autogenerated and we only specify `@param {(String|Number)} [organization]` in the comments.
PS: You can see the same pattern being used in `application.getAll([options])`.

Comment (sradevski) [reply DOCUMENTATION.md:6053]: I did a quick search and I didn't find one, but I guess my searching skills failed me :)

**Comment by thgreasi** [comment]
@balena-ci rebase

**Pull request merged**